types/weixin-app/weixin-app-tests.ts

```diff
@@ -0,0 +1,3 @@
+import {} from 'weixin-app';
+
+getCurrentPages();
```

Comment:
What is your point?
Where did What come from?
If you mean the test code `getCurrentPages`, that's just a test to make sure the dts could be used.
@RyanCavanaugh